### PR TITLE
[main][deploy] preparing script for deployment xalpaca on ftm

### PR DESCRIPTION
**Pull request by blackalpacaa**
## Description

- Create script for setting config at workers03.
- Create script for add pool and set pool for MiniFL

## Why?

- preparing script for deployment xalpaca on ftm

### CODE REVIEW

File: deploy/exec/minifl/config/set-pool-no-timelock.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+  */
+  const UPDATES: Array<IUpdate> = [
+    {
+      STAKING_TOKEN: "fdAlpacaToken",
```
Comment (alpanaca) [inline deploy/exec/minifl/config/set-pool-no-timelock.ts:36]:
```suggestion
      STAKING_TOKEN: "fdALPACA",
```

File: deploy/exec/workers/config/set-beneficial-vault-config-workers03.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    "BTC-WFTM SpookyWorker",
+    "ALPACA-WFTM SpookyWorker",
+    "ETH-WFTM SpookyWorker",
+    "BOO-WFTM SpookyWorker",
+    "USDC-WFTM SpookyWorker",
+    "WFTM-TOMB TombWorker",
+    "TSHARE-WFTM TombWorker",
+    "TOMB-WFTM TombWorker",
+  ];
+  const rewardPathInput: Array<string> = ["CAKE", "BUSD", "ALPACA"];
```
Comment (alpanaca) [inline deploy/exec/workers/config/set-beneficial-vault-config-workers03.ts:47]: Be careful with "CAKE". There won't be CAKE in FTM.

**Review by alpanaca — CHANGES_REQUESTED** [review]

File: deploy/exec/workers/config/set-beneficial-vault-config-workers03.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  const BENEFICIAL_VAULT_BOUNTY_BPS = "5263";
+  const BENEFICIAL_VAULT_ADDRESS = "0x44B3868cbba5fbd2c5D8d1445BDB14458806B3B4";
```
Comment (alpanaca) [inline deploy/exec/workers/config/set-beneficial-vault-config-workers03.ts:49]: Remove this as we need to wait for Feeder to go up.

File: deploy/exec/minifl/config/set-pool-no-timelock.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  const config = getConfig();
+  const deployer = (await ethers.getSigners())[0];
+  const inputs: Array<IInput> = [];
+  const miniFL = MiniFL__factory.connect(config.MiniFL!.address, deployer);
+  let nonce = await deployer.getTransactionCount();
+
+  /// @dev derived input
+  for (let i = 0; i < UPDATES.length; i++) {
+    const pool = config.MiniFL!.pools.find((p) => p.stakingToken == UPDATES[i].STAKING_TOKEN);
```
Comment (sora-alpaca) [inline deploy/exec/minifl/config/set-pool-no-timelock.ts:51]: if use ! then MiniFL is undefined, this line will be thrown error

u can use `config.MiniFL?.pools.find(...)` then pool will return as undefined if MiniFL is `undefined`

File: deploy/exec/minifl/config/add-pool-no-timelock.ts [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      ALLOC_POINT: 0,
+      REWARDER_ADDRESS: "0x0000000000000000000000000000000000000000",
+      IS_DEBT_TOKEN: false,
+      WITH_UPDATE: true,
+    },
+  ];
+
+  const config = getConfig();
+  const deployer = (await ethers.getSigners())[0];
+  const miniFL = MiniFL__factory.connect(config.MiniFL!.address, deployer);
```
Comment (sora-alpaca) [inline deploy/exec/minifl/config/add-pool-no-timelock.ts:38]: what happen if MiniFL is 0x00 or undefined

**Review by alpanaca — APPROVED** [review]

**Review by jr-alpaca — APPROVED** [review]

**Pull request merged**